src/wire/ndisc.rs

```diff
@@ -228,7 +228,7 @@ impl<'a> Repr<'a> {
                     let opt = NdiscOption::new_checked(packet.payload())?;
                     match opt.option_type() {
                         NdiscOptionType::SourceLinkLayerAddr => Some(opt.link_layer_addr()),
-                        _ => { return Err(Error::Unrecognized); }
+                        _ => {}
```

Comment:
Very minor nit: we use `()` for the do-nothing case. r=me once this is fixed.

Reply:
Fixed!